### PR TITLE
Fix project item hover

### PR DESCRIPTION
**Pull request by olgatim**

### CODE REVIEW

File: src/scenes/Projects/components/List/components/ProjectScene/ProjectScene.module.scss [hunk 1]
```diff
@@ -45,6 +41,12 @@
   }
 }
 
+.hovered {
+  & > a > div {
```
Comment (lazio) [inline src/scenes/Projects/components/List/components/ProjectScene/ProjectScene.module.scss:45]: can we use class selectors instead of tags?

**Comment by lazio** [comment]
hover shouldn't be triggered on a free space https://tppr.me/6sbYf

**Comment by lazio** [comment]
ok, there is a free space where hover should be triggered:)
between lines https://tppr.me/OAxO7
Can we fix that?

**Comment by lazio** [comment]
@olgatim

**Comment by Aleksandr-Tyagun** [comment]
The hover effect still won't trigger on these areas http://joxi.ru/E2p7DWPhGj6Onm

**Pull request merged**